packages/html-spec/src/loader/loader.json

```diff
@@ -0,0 +1,143 @@
+[
+    {
+        "component": "loader",
+        "selector": [
+            ".k-loader"
+        ],
+        "options": [
+            {
+                "name": "size",
+                "values": [
+                    {
+                        "name": "small",
+                        "selector": [
+                            ".k-loader-sm"
+                        ]
+                    },
+                    {
+                        "name": "medium",
+                        "selector": [
+                            ".k-loader-md"
+                        ]
+                    },
+                    {
+                        "name": "large",
+                        "selector": [
+                            ".k-loader-lg"
+                        ]
+                    }
+                ]
+            },
+            {
+                "name": "themeColor",
+                "values": [
+                    {
+                        "name": "base",
+                        "selector": [
+                            ".k-loader-{options.themeColor}"
```

Comment:
replace options.themeColor with the actual themeColor here for each value: k-loader-base, k-loader-primary, etc.